lib/xilinx/xc7series/crc_test.cc

```diff
@@ -0,0 +1,13 @@
+#include <prjxray/xilinx/xc7series/crc.h>
+
+#include <gtest/gtest.h>
+
+namespace xc7series = prjxray::xilinx::xc7series;
+
+TEST(IcapCrcTest, SimpleTests) {
+	EXPECT_EQ(xc7series::icap_crc(0, 0, 0), (uint32_t)0x0);
+	EXPECT_EQ(xc7series::icap_crc(~0, ~0, 0), 0xBF86D4DF);
```

Comment:
What code path is this testing?  Why is ~0 a more relevant test compared to all other possible values?